src/gui/menus/RecentProjectsMenu.cpp

```diff
@@ -0,0 +1,74 @@
+#include "RecentProjectsMenu.h"
+#include "embed.h"
+#include "ConfigManager.h"
+#include <QFileInfo>
+#include "Song.h"
+#include "Engine.h"
+#include "MainWindow.h"
+#include "GuiApplication.h"
```

Comment:
Minor, but we usually pay attention to the order of the included headers. You can take a look at other files for examples. This one could be improved to:

```cpp
#include "RecentProjectsMenu.h"

#include <QFileInfo>

#include "ConfigManager.h"
#include "Engine.h"
#include "Song.h"

#include "embed.h"
#include "GuiApplication.h" 
#include "MainWindow.h"
``` 

Bonus points for grouping LMMS headers by GUI/Core as in the suggestion above, but that's often difficult because it's not always separated clearly.

Reply:
Thanks—I have switched to the correct order now.